app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -40,13 +33,28 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               label="Title"
               name="title"
               className="w-full flex-grow-0"
+              placeholder="Add Title"
               required
             />
-            <Textarea
+            <Input
               label="Description"
               name="description"
               className="w-full flex-grow-0"
-              rows={8}
+              placeholder="Add Description"
+              required
```

Comment:
https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Declaring%20boolean%20props

Reply:
I guess this is correct as per book. have just added boolean variable.

Reply:
@naufil50 Not quite. You might've missed the last sentence:
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/31244999/184885554-ee2800c9-5e2a-4994-b4c5-8e2a88d95aa3.png">